app/questionnaire_state/question.py

```diff
@@ -1,6 +1,23 @@
+from app.questionnaire_state.answer import Answer
+from app.questionnaire_state.item import Item
 
 
-class Question(object):
+class Question(Item):
     def __init__(self, id):
-        self.id = id
+        super().__init__(id=id)
         self.answers = []
+        self.children = self.answers
+
+    def get_answers(self):
+        answers = []
+        for child in self.children:
+            answers.append(child.get_answer())
```

Comment:
At the question level it is .append rather than .extend as it's calling child.get_answer() rather than child.getanswers().

On reflection this isn't clear and maybe returning a list containing a single element would be clearer, keeping the interface clean and allowing this class to use the parent method.